CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,2 @@
+Add missed global config env vars to docker (IOTA_CONFIG_RETRIEVAL, IOTA_DEFAULT_KEY, IOTA_DEFAULT_TRANSPORT)
```

Comment:
"docker" is mentioned in this CNR entry, but the PR seems not related with docker itself but with basic env vars support...

Reply:
My mistake. I'm always thinking in docker when see env vars.